chia/plot_sync/receiver.py

```diff
@@ -82,6 +83,7 @@ class Receiver:
     _keys_missing: List[str]
     _duplicates: List[str]
     _total_plot_size: int
+    _total_effective_plot_size: float
```

Comment:
it seems odd to make this a `float`. is there a good reason for that?
when adding these up, you may have non-deterministic results

Reply:
Right, I changed this to `int`.